types/src/transaction/mod.rs

```diff
@@ -476,10 +476,13 @@ pub struct SignedTransaction {
     /// Prevents serializing the same authenticator multiple times to determine size.
     #[serde(skip)]
     authenticator_size: OnceCell<usize>,
+
+    /// A cached hash of the transaction.
+    #[serde(skip)]
+    committed_hash: OnceCell<HashValue>,
```

Comment:
can we make the clone work for all the once cell too? I think we do clone txns across multiple places

Reply:
I checked the OnceCell implementation, the value is cloned